css/MLS.css

```diff
@@ -87,7 +87,7 @@ a:hover { text-decoration: underline; }
  * term ends and the links begin.
  */
 .ltx_indexrefs > .ltx_text:first-child:before {
-  content: " –";
+  content: " â€“";
```

Comment:
I guess this needs an explaining comment?

Reply:
It was a mis-take.

Reply:
Hmm... I believe I now understand this mistake.

It wasn't as a I first thought that I pressed some weird key, but that master doesn't say code " -" as space minus sign, but as 0x20 0xe2 0x80 0x93, which is space en-dash, where en-dash is the Unicode character U+2013
For some reason my editor must have completely stumbled on this.

Reply:
Looks good now.  Resolving.